k9mail/src/main/java/com/fsck/k9/mailstore/LocalMessage.java

```diff
@@ -27,18 +27,16 @@
 
 
 public class LocalMessage extends MimeMessage {
-    protected MessageReference mReference;
     private final LocalStore localStore;
 
-    private long mId;
-    private int mAttachmentCount;
-    private String mSubject;
-
-    private String mPreview = "";
-
-    private long mThreadId;
-    private long mRootId;
+    private long databaseId;
```

Comment:
I'd like `id` in `LocalMessage` and `LocalFolder` classes to mean the database ID. When referencing these values from external classes it's `messageId` and `folderId` respectively.